test/helpers.js

```diff
@@ -24,7 +24,7 @@ export const defaultPaginationResult = {
   totalPage: 2,
 };
 
-export const itReturnsTheRightPaginationValue = (endpoint, user) => {
+export const itReturnsTheRightPaginationValue = (endpoint, method, user, data = {}) => {
```

Comment:
Nice work here. 

One of the best practices for parameters is to make them an object parameter when they are 3. Let's make the parameters an object to make it easier to remember and be consistent. I don't mind leaving it like this but I am afraid that it will not be consistent with `itReturnsAnErrorForInvalidToken` which now has 6 parameters. We might even update it later to accommodate more edge cases. 

To fix this and future problems, let's make this and other functions an object parameter. That way this function is easier to maintain and update before the place we have used them become too much to update.

```suggestion
export const itReturnsTheRightPaginationValue = ({endpoint, method, user, data}) => {
```